charts/jenkins/README.md

```diff
@@ -149,15 +149,22 @@ In order to upgrade, [uninstall](#uninstall-chart) the Jenkins Deployment before
 See [Customizing the Chart Before Installing](https://helm.sh/docs/intro/using_helm/#customizing-the-chart-before-installing). To see all configurable options with detailed comments, visit the chart's [values.yaml](./values.yaml), or run these configuration commands:
 
 ```console
-# Helm 2
-$ helm inspect values jenkinsci/jenkins
-
 # Helm 3
 $ helm show values jenkinsci/jenkins
+
+# Helm 2
+$ helm inspect values jenkinsci/jenkins
 ```
 
+For a summary of all configurable options, see [VALUES_SUMMARY.md](./VALUES_SUMMARY.md)
```

Comment:
restoring the parameter tables in a new file at least until `values.yaml` gets cleaned up